common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -132,6 +133,7 @@ public static void submitSilent(@SuppressWarnings(WarningType.rawtype_warning) @
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     public static FinalizableResultFuture<CommandResult> submitSilentReturningFuture(@SuppressWarnings(WarningType.rawtype_warning) @NonNull final BaseCommand command) {
 
+        CodeMarkerManager.markCode(10011);
```

Comment:
Can we put these marker ids in a constants file?

Reply:
Done. Thanks for the suggestion.